modules/calcite/src/test/sql/subquery/scalar/test_scalar_subquery.test

```diff
@@ -101,9 +101,23 @@ SELECT a, (SELECT SUM(b) FROM test tsub WHERE test.a=tsub.a) FROM test
 12	21.000000
 13	22.000000
 
-query II
+statement error
```

Comment:
Why statement error? The result seems ok.

Reply:
Hm. Don't know why this test was marked as `statement error`. I don't see any problem with this query too. 
Fixed, thanks.